packages/flutter/lib/src/material/text_field.dart

```diff
@@ -759,16 +759,20 @@ class _TextFieldState extends State<TextField> implements TextSelectionGestureDe
         && effectiveDecoration.counterText == null
         && widget.buildCounter != null) {
       final bool isFocused = _effectiveFocusNode.hasFocus;
-      counter = Semantics(
-        container: true,
-        liveRegion: isFocused,
-        child: widget.buildCounter(
-          context,
-          currentLength: currentLength,
-          maxLength: widget.maxLength,
-          isFocused: isFocused,
-        ),
+      final Widget buildCounter = widget.buildCounter(
```

Comment:
I think we should call this `counterChild` or `builtCounter` or something like that.  That way it's distinguished from the function buildCounter.

Reply:
I'll take your suggestion. 😉